ontologies/vocabs.ttl

```diff
@@ -1055,6 +1062,265 @@ Specifies an endpoint machine used for work by an organization that needs protec
 	skos:prefLabel "workstation"^^xsd:string ;
 	.
 
+gist:_MalwareCapability_Accesses-Remote-Machines
+	a gist:MalwareCapability ;
+	skos:definition """STIX 2.1 description:
+Indicates that the malware instance or family is able to access one or more remote machines."""^^xsd:string ;
+	skos:prefLabel "accesses-remote-machines"^^xsd:string ;
```

Comment:
not very human readable, remove dashes? Maybe same case, e.g. "Accesses Remote Machines"